tests/baselines/reference/user/async.log

```diff
@@ -43,8 +43,8 @@ node_modules/async/auto.js(159,18): error TS2695: Left side of comma operator is
 node_modules/async/auto.js(159,50): error TS2695: Left side of comma operator is unused and has no side effects.
 node_modules/async/autoInject.js(44,17): error TS2695: Left side of comma operator is unused and has no side effects.
 node_modules/async/autoInject.js(134,6): error TS2695: Left side of comma operator is unused and has no side effects.
-node_modules/async/autoInject.js(136,25): error TS2722: Cannot invoke an object which is possibly 'undefined'.
```

Comment:
@weswigham these are all ordering issues, i thought your last change should address these..

Reply:
Didn't accept any baselines, was waiting for the bot to open a PR.